requirements.txt

```diff
@@ -2,7 +2,7 @@ Flask==0.10.1
 Flask-Login==0.2.11
 Flask-Script==2.0.5
 Flask-WTF==0.11
-git+https://github.com/alphagov/digitalmarketplace-utils.git@1.0.6#egg=digitalmarketplace-utils==1.0.6
+git+https://github.com/alphagov/digitalmarketplace-utils.git@2.0.4#egg=digitalmarketplace-utils==2.0.4
```

Comment:
I don't really understand how the tests are passing, but version 2.0.4 of utils doesn't have the `get_selection_answers` method as far as I can see, and when running the app the declaration page gives me 
```
File "/Users/kevinkeenoy/gdsworkspace/digitalmarketplace-supplier-frontend/app/main/views/frameworks.py", line 49, in framework_supplier_declaration
response = data_api_client.get_selection_answers(
AttributeError: 'DataAPIClient' object has no attribute 'get_selection_answers'
```